### ISSUE TITLE
Purchase Orders, Referrals, and a Question.

### ISSUE THREAD

**Issue by ktdmedia** [drhenner/ror_ecommerce#152]
Hey there ror_ecommerce contributors,

I just started playing with the system and first off let me say, great work! It's going to save me a ton of time with a project I'm currently working on. 

I've ran into a few error over the last few days, they may have been something I've done but, I didn't tweak much from the master build aside from a couple menu styles.

Purchase Orders don't save products when you click the New Variant button and enter the required details. After clicking update it doesn't save. This occurs if the PO is new or already created. 

Referral system has no way to set referral types. Also, when creating a new referral manually it doesn't save the referring user's email when saving either.  

Last thing, Why Foundation? Is there a reason for Foundation over Bootstrap or is it just a preference? I personally think Bootstrap has really evolved past Foundation. Would it be opposed to converting ror_e to Bootstrap? I would be willing to contribute to the recoding if so. I started a version the other day and it's definitely doable. 

Thanks!

**Comment by drhenner** [comment]
I'll start looking at this on the weekend.

I'm not for or against Bootstrap.  Someone volunteered to add Foundation and I said yes.  If you want to move things to Bootstrap I'd start with the customer-facing site.  I'd take any / all contributions.

**Comment by ktdmedia** [comment]
Hey there, 

Any updates?

I'll start working on it and post something by this weekend.

**Comment by drhenner** [comment]
I set the referral types  in the seeds.  I had made the commit just after you posted.  

As for bootstrap.  I'd like to have the customer facing site using bootstrap just keep ERB.  I actually like both ERB & HAML but everyone knows ERB and some some don't know HAML.  Plus there are ERB to HAML scripts.

I can try out the Purchase Order.  Did you actually go to Admin::Inventory::Receiving to receive the purchase order?  That is required for the inventory to be added.  (I haven't manually tested yet)
